tests/sce/script_tester.py

```diff
@@ -52,18 +52,18 @@
 xccdf_result_map["XCCDF_RESULT_FIXED"] = 109
 
 xccdf_reverse_result_map = dict()
-for key, value in xccdf_result_map.iteritems():
+for key, value in xccdf_result_map.items():
     xccdf_reverse_result_map[value] = key
 
 # set all XCCDF result types as environment variables
 # (this is especially useful for bash scripts)
-for key, value in xccdf_result_map.iteritems():
+for key, value in xccdf_result_map.items():
     os.environ[key] = str(value)
 
 import subprocess
 
 for file_path in files:
     result = subprocess.call(["./%s" % (file_path)])
-    result_text = xccdf_reverse_result_map[result] if xccdf_reverse_result_map.has_key(result) else "Unknown exit code %i" % (result)
```

Comment:
This is perfectly valid, but I personally dislike the non-trivial form ternary operators, as it is difficult to see what is going on (as opposed to an if statement). I don't insist on changing it, but you may give it a second thought.

Reply:
I like this suggestion very much. I will change it.